packages/sentry/.eslintignore

```diff
@@ -0,0 +1,10 @@
+dist
+node_modules
+.eslintrc.js
+jest.config.js
+dist
+lib
+src/lib/data.ts
+src/lib/index.ts.ts
+src/lib/data
+src/templates
```

Comment:
you can remove all these lines and just keep `lib`

Reply:
okay